src/models.py

```diff
@@ -269,6 +277,16 @@ class LabelPayloadData(PartialModelMixin, BaseModel):
 PartialLabelPayloadData = LabelPayloadData.model_as_partial()
 
 
+class LabelObject(BaseModel):
+	dataset_id: int
+	user_id: str
+	file_type: str
+	file_alias: str
+	file_path: str
+	label_description: str
+	audited: bool
```

Comment:
same here. I would suggest we do not call this LabelObject or alternatively rename the existing labels. This seems quite confusing to me like this. 

Other possible words we could use for this (or the other) are: Annotation, Tag,

Maybe also add a docstring to this one, so that the difference is documented.

Reply:
UserLabelObject ?

Reply:
That's better. Maybe UserLabelFile, or something? @cmosig might have a clever idea here.

Reply:
maybe RawLabelFile or UploadedLabelFile. UserLabelFile ist also fine.